.github/workflows/zowe-cli-deploy-all.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Use Node.js 16
       uses: actions/setup-node@v3
```

Comment:
Should this be updated to use the `lts/*` version of node?
CC: @t1m0thyj 

Note: this same comment may apply to multiple steps in this workflow (or other workflows)

Reply:
For the deployment pipelines I kind of like sticking with something we know works - not sure if jumping to Node 18 and npm 9 is worth the risk of breakage. But I know in most of our repos we try to use the latest LTS version whenever possible, so I'm open to doing that here too 🙂